ProcessLib/ProcessVariable.h

```diff
@@ -52,7 +52,7 @@ class ProcessVariable
     template <typename OutputIterator, typename GlobalSetup, typename ...Args>
     void createNeumannBcs(OutputIterator bcs,
         MeshGeoToolsLib::BoundaryElementsSearcher& searcher,
-        GlobalSetup const& global_setup,
+        GlobalSetup const&,
```

Comment:
This is used to infer the `GlobalSetup` type using a mechanism called "argument dependent lookup".
I can not avoid this because I'd need to specify the unknown `Args` types in the process in the [function call](https://github.com/ufz/ogs/blob/master/ProcessLib/GroundwaterFlowProcess.h#L150-L156).